src/pagination.rs

```diff
@@ -1,151 +1,3 @@
-//! A module providing pagination capabilities.
-use super::*;
+//! (TODO) A module providing pagination capabilities.
```

Comment:
Why have you removed the pagination? I generally recommend deleting parts of the system, once they're actually ready to be replaced (when there is an alternative solution). Plus you didn't really make a breaking change to the way the pagination endpoints are written by adding support for Acril, so I don't see a reason.

Reply:
This is the first commit, mind you

Reply:
It says "T O D O", don't you see?

Reply:
Done.